lib/src/tabs/Tabs.tsx

```diff
@@ -67,24 +62,21 @@ const DxcTabs = ({
           variant="scrollable"
           scrollButtons="auto"
         >
-          {tabs.map((tab, i) => {
-            const tabContent = React.forwardRef((props, ref) => <div role="button" {...props} ref={ref} />);
-            return (
-              <Tab
-                tabIndex={(activeTabIndex === i || innerActiveTabIndex === i) && !tab.isDisabled ? tabIndex : -1}
-                key={`tab${i}${tab.label}`}
-                label={getLabelForTab(tab)}
-                disabled={tab.isDisabled}
-                disableRipple={true}
-                onMouseEnter={() => {
-                  onTabHover(i);
-                }}
-                onMouseLeave={() => {
-                  onTabHover(null);
-                }}
-              />
-            );
-          })}
+          {tabs.map((tab, i) => (
+            <Tab
+              tabIndex={(activeTabIndex === i || innerActiveTabIndex === i) && !tab.isDisabled ? tabIndex : -1}
+              key={`tab${i}${tab.label}`}
+              label={getLabelForTab(tab)}
+              disabled={tab.isDisabled}
+              disableRipple={true}
+              onMouseEnter={() => {
+                onTabHover(i);
```

Comment:
When an event is marked as optional in _types.ts_, we should add optional method chaining when it is called: `onCloseClick?.()` or it will throw a **ReferenceError**. Be careful when removing `typeof === "function"`, this line is also controlling that is not undefined or null.